kratos/utilities/geometry_utilities/brep_trimming_utilities.h

```diff
@@ -0,0 +1,410 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+
+#if !defined(KRATOS_BREP_TRIMMING_UTILITIES_H_INCLUDED)
+#define KRATOS_BREP_TRIMMING_UTILITIES_H_INCLUDED
+
+// System includes
+#include "includes/define.h"
+
+// External includes
+#include "clipper/clipper.h"
```

Comment:
I strongly suggest to only include this in a cpp, as otherwise this will be propagated to lots of places.
I see you templated the code but in this case I would pay the price of instantiating the required classes